impls/bcpl/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:18.04
+MAINTAINER Joel Martin <github@martintribe.org>
+
+##########################################################
+# General requirements for testing or common across many
+# implementations
+##########################################################
+
+RUN apt-get -y update
+
+# Required for running tests
+RUN apt-get -y install make python
+
+# Some typical implementation and test requirements
+RUN apt-get -y install curl libreadline-dev libedit-dev
+
+RUN mkdir -p /mal
+WORKDIR /mal
+
+##########################################################
+# Specific implementation requirements
+##########################################################
+
+RUN apt-get -y install gcc mesa-common-dev libglu1-mesa-dev freeglut3-dev
+
+RUN cd /usr/local && \
+    curl -o bcpl.tgz https://www.cl.cam.ac.uk/~mr10/BCPL/bcpl.tgz && \
+    tar zxf bcpl.tgz && chmod -R a+rX BCPL
```

Comment:
You could delete bcpl.tgz as a final step to shrink this layer.

Reply:
Ah yes. Too long spent writing Ansible playbooks for VMs, where a few magabytes either way don't really matter, I'm afraid.

Reply:
Should be fixed by 0149fc6.